src/viam/examples/modules/complex/proto/buf.lock

```diff
@@ -4,5 +4,4 @@ deps:
   - remote: buf.build
     owner: googleapis
     repository: googleapis
-    commit: ee48893a270147348e3edc6c1a03de0e
-    digest: shake256:a35b0576a2b55dad72747e786af05c03539c2b96be236c9de39fe10d551931ac252eb68445c0cef6bbd27fa20e8c26eee5b8a9fe9c2fde6f63a03e18f8cf980d
+    commit: 4ed3bc159a8b4ac68fe253218760d035
```

Comment:
This file updated automatically at some point. I don't know what caused it, but I've put the change in a separate commit so it's easy to revert if it's not supposed to be here.